package.json

```diff
@@ -107,7 +107,7 @@
     "@types/react": "18.3.1",
     "@types/react-dom": "18.3.0",
     "postcss": "^8.3.11",
-    "jackspeak": "2.3.6"
+    "jackspeak": "2.1.1"
```

Comment:
Why reduce the version?

Reply:
The newer version doesn't work with Yarn 1.